moveit_setup_assistant/templates/moveit_config_pkg_template/launch/ros_controllers.launch

```diff
@@ -6,12 +6,6 @@
 
   <!-- Load the controllers -->
   <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
-    output="screen" ns="[ROBOT_NAME]" args="[ROS_CONTROLLERS]"/>
-
-  <!-- Convert joint states to TF transforms for rviz, etc -->
-  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
-    respawn="false" output="screen">
-    <remap from="/joint_states" to="[ROBOT_NAME]/joint_states" />
-  </node>
```

Comment:
why remove this?

Reply:
There is already a `robot_state_publisher` node in `demo.launch`.
removing the `namespace` is for the reason explained [here](https://github.com/ros-planning/moveit/pull/1051/files#diff-b722193e66bd85583ac56b9d68186447L763)